man/systemd.network.xml

```diff
@@ -1121,6 +1121,41 @@ Table=1234</programlisting></para>
           Defaults to <literal>no</literal>.</para>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetLabel=</varname><replaceable>label</replaceable></term>
+        <listitem>
+
+          <para>This setting provides a method for integrating dynamic network configuration into Linux
+          <ulink url="https://docs.kernel.org/netlabel/index.html">NetLabel</ulink> subsystem rules, used by
+          <ulink url="https://en.wikipedia.org/wiki/Linux_Security_Modules">Linux security modules
+          (LSMs)</ulink> for network access control. The option expects a single NetLabel label. The label
+          must conform to lexical restrictions of LSM labels. When an interface is configured with IP
+          addresses, the addresses and subnetwork masks will be appended to the <ulink
+          url="https://github.com/SELinuxProject/selinux-notebook/blob/main/src/network_support.md">NetLabel
+          Fallback Peer Labeling</ulink> rules. They will be removed when the interface is
+          deconfigured. Failures to manage the labels will be ignored.</para>
```

Comment:
This should really explain what this all means. i.e. why one would do this.

i.e. write this for somebody who is *not* at home in the NetLabel world, for example me.

Reply:
also, this doesn't mention whether this is about ingress or egress

Reply:
This is a new option to complement rather advanced setup with NetLabels, so this will never be a beginner level option, but I tried to put more descriptive text here. About ingress, I'm not sure how Smack uses NetLabels and if it has the same limitations but at least for SELinux the checks are only for receiving side.